.vib/moodle/goss/moodle.yaml

```diff
@@ -0,0 +1,14 @@
+file:
+  /opt/bitnami/apache/conf/vhosts/moodle-vhost.conf:
+    exists: true
+    contains:
+    {{ range $rule := .Vars.apache.rules }}
+      - {{ $rule }}
+    {{ end }}
+  /opt/bitnami/php/etc/php.ini:
+    exists: true
+    contains:
+      # Checking if the php.ini keys and pqsql extension are set
+      - extension = pgsql
+      - /^memory_limit/
+      - /^max_input_vars/
```

Comment:
I would add a check for the `user` and `group` because of `ensure_user_exists "$WEB_SERVER_DAEMON_USER" --group "$WEB_SERVER_DAEMON_GROUP"`:

```
group:
  daemon:
    exists: true
user:
  daemon:
    exists: true
```